src/main/java/com/monumental/controllers/SearchController.java

```diff
@@ -37,7 +44,10 @@
      * @param longitude - The longitude of the comparison point
      * @param distance - The distance from the comparison point to search in, units of miles
      * @param sortType - The way in which to sort the results by
+     * @param onlyActive - If true, only active monuments will be searched. If false, both inactive and active will be searched
      * @return            Matching Monuments based on their title, artist or location
+     * @throws AccessDeniedException - If trying to search for inactive monuments without being a partner or researcher
```

Comment:
Can you correct this to be `If trying to search for inactive monuments without being a partner or above`?